wazo_confgend/generators/extensionsconf.py

```diff
@@ -171,20 +172,22 @@ def generate(self, output):
             'vmusermsg',
         )
         extenfeatures = asterisk_conf_dao.find_extenfeatures_settings(features=extenfeature_names)
-        xfeatures = {extenfeature.typeval: {'exten': extenfeature.exten,
-                                            'commented': extenfeature.commented}
-                     for extenfeature in extenfeatures}
+        xfeatures = {
+            extenfeature.typeval: {
+                'exten': extenfeature.exten, 'commented': extenfeature.commented
+            } for extenfeature in extenfeatures
+        }
 
         # foreach active context
         for ctx in asterisk_conf_dao.find_context_settings():
             # context name preceded with '!' is ignored
-            if conf and conf.has_section('!%s' % ctx['name']):
+            context_name = ctx['name']
+            if conf and conf.has_section('!%s' % context_name):
                 continue
 
-            print >> options, "\n[%s]" % ctx['name']
```

Comment:
The space between `usersharedlines` and the first context is lost
Can you add an empty line between the loop? It's only to keep generated file easily readable